projects/eslint-plugin/rules/general/lib/rules/prefer-length-check.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * SPDX-FileCopyrightText: © 2021 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const message = 'prefer using .length instead of .length === 0';
+
+module.exports = {
+	create(context) {
+		return {
+			BinaryExpression(node) {
+				if (node.operator) {
+					if (
+						node.operator === '===' ||
+						node.operator === '<' ||
+						node.operator === '>'
```

Comment:
I think we would also need to check what the operator is comparing against. We only would want to check when explicitly checking against `0`


```js
// valid
if (foo.length > 5) {
}
```